pyproject.toml

```
@@ -24,11 +24,17 @@ classifiers = [
     "Programming Language :: Python :: 3.12",
 ]
 dependencies = [
-    "numpy",
+    "pydantic-settings",
     "gymnasium==0.29.*",
+    "bokeh",
+    "loguru",
     "matplotlib",
-    "stable-baselines3>=2.3.2",
+    "numpy",
+    "sb3-contrib",
     "scipy",
+    "stable-baselines3>=2.3.2",
+    "tqdm",
+    "tomlkit",
```

any reason you switched up the order? I got confused because I thought you removed numpy for a moment.

If anything, I'd argue let's keep them alphabetical?

```suggestion
    "bokeh",
    "gymnasium==0.29.*",
    "loguru",
    "matplotlib",
    "numpy",
    "pydantic-settings",
    "sb3-contrib",
    "scipy",
    "stable-baselines3>=2.3.2",
    "tomlkit",
    "tqdm",
```

---

They were unsorted to begin with (cf. v. 1.0.8 - still not sorted). But I completely agree that they should be! 🙂